packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -367,7 +389,6 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
 --media-width-sm: 544px;
```

Comment:
can these `--media-width-*` vars be used? seems like they'd only appear in media queries but you can't use css vars in media queries

Reply:
You're right. They might not be super useful, but at least users have the values to reference? I don't know.